freedomofpress/securedrop
# [0.7.0] Bump to 0.7.0~rc3

## Status

Ready for review
## Description of Changes

Bumps 0.7 branch to 0.7.0~rc3, using the [release management guide](https://docs.securedrop.org/en/latest/development/release_management.html)

## Testing

* Ensure the release notes are accurate and reflect the latest changes merged to the 0.7 branch
* Ensure the release management instructions have been properly applied
